packages/api-folders/src/context/entries.ts

```diff
@@ -0,0 +1,166 @@
+/**
+ * Package mdbid does not have types.
+ */
+// @ts-ignore
+import mdbid from "mdbid";
+/**
+ * Package @commodo/fields does not have types.
+ */
+// @ts-ignore
+import { string, withFields } from "@commodo/fields";
+import WebinyError from "@webiny/error";
+import { NotFoundError } from "@webiny/handler-graphql";
+import { validation } from "@webiny/validation";
+
+import { FoldersConfig, Entry, ListEntriesParams, EntryInput, IEntries } from "~/types";
+
+const CreateDataModel = withFields({
+    id: string({ validation: validation.create("required") }),
+    folderId: string({ validation: validation.create("required") }),
+    tenant: string({ validation: validation.create("required") }),
+    locale: string({ validation: validation.create("required") })
+})();
```

Comment:
We want to retire `commodo`, so let's use this opportunity and switch to Joi. You can find an example here: https://github.com/webiny/webiny-js/blob/next/packages/api-mailer/src/crud/mailer/onBeforeSend.ts#L9